packages/components/src/ErrorState/ErrorState.js

```diff
@@ -11,11 +11,12 @@ import {
     EmptyStateBody,
     Button
 } from '@patternfly/react-core';
+import './filter-chips.scss';
```

Comment:
This file does not exists.
```suggestion
import './error-state.scss';
```